.github/workflows/ci.yaml

```
@@ -29,9 +29,9 @@ jobs:
       run: |
         python docs/bin/clone-core.py
 
-    - name: Run docs-build Sanity
+    - name: Run docs-build Checkers
```

Why not use lower-case here? (same below)
```suggestion
    - name: Run docs-build checkers
```

---

I was following the existing convention, title case.